scripts/python/phdf_diff.py

```diff
@@ -204,6 +276,9 @@ def compare(files, all=False, brief=True, quiet=False, one=False, tol=1.0e-12):
         if one and not same:
             break
 
+    if not no_meta_variables_diff:
+        return(8)
```

Comment:
Are the return values defined somewhere in the documentation or am I simply missing them.

Reply:
It looks like these return values were made sequential starting from 1, using the standard return value of 0 to the command line to indicate no errors. I don't think the nonzero return values are treated specially anywhere. Would enumerating the return values at the top of the file be sufficient?

Reply:
Yes, I think so simply having a summary in the docstring of what each exit code means would be nice so you don't have to go digging through the logic.